packages/legend-query-builder/src/components/__tests__/QueryBuilderMilestoning.test.tsx

```diff
@@ -393,19 +393,31 @@ describe(
           expectedNumberOfPropertyParameterValues,
         );
 
-        // Check if we have paramter panel opened and able to run query
+        // Check if we have parameter panel opened and able to run query
         if (expectedNumberOfParameterValues > 0) {
           expect(queryBuilderState.showParametersPanel).toBe(true);
           await waitFor(() =>
             renderResult.getByTestId(
               QUERY_BUILDER_TEST_ID.QUERY_BUILDER_RESULT_PANEL,
             ),
           );
-          fireEvent.click(renderResult.getByText('Run Query'));
-          const executeDialog = await waitFor(() =>
-            renderResult.getByRole('dialog'),
-          );
-          expect(getByText(executeDialog, 'Set Parameter Values'));
+          const hasNonMilestoningParams =
```

Comment:
lets not hardcode `hasNonMilestoningParams` this into the test. lets follow the conventions of the test and include as a test property parameter